src/WebJobs.Script.WebHost/Models/ActivityBase.cs

```diff
@@ -1,14 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-
-namespace Microsoft.Azure.WebJobs.Script.WebHost.Models
-{
-    public class ActivityBase
```

Comment:
The contract is shared between CV1 and Flex. I moved these existing types out into the new Microsoft.Azure.Functions.Platform.Metrics.LinuxConsumption library